src/fsharp/est.fs

```diff
@@ -17,6 +16,12 @@ module internal ExtensionTyping =
     open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library // frontAndBack
     open Internal.Utilities.FileSystem
 
+#if FX_RESHAPED_REFLECTION
+    open Microsoft.FSharp.Core.ReflectionAdapters
+#else
+    type BindingFlags = System.Reflection.BindingFlags
```

Comment:
Why do this and remove `open System.Reflection`, requiring qualification in other places?  Just leave in `open System.Reflection` should work (maybe need to move it down).